### PR TITLE
Project outline for #impartial-hedgehog 

### PR DESCRIPTION
**Pull request by alfonsotech**
Project outline for #impartial-hedgehog

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ Philosophy Learners Guild is a website that allows people who are interested in
 This is a refactor for a project created on the front end. I would like to add a backend (SQL database) to the project with CRUD; set up the project as an open source project that includes a project wiki; create a wireframe for scaling out the project from a basic curriculum site to one that includes the ability for users to register, log-in, and interact with other users. 
 
 ###Developer Stories
-- [ ] Developer will want to make decisions about how best to architecture the site
-- [ ] Developer will want to make decisions about fonts and color schema to be used - light and airy is the aesthetic. They will research and find a website with this aesthetic and lift the fonts and colors as a base for this.
+- [ ] Developer will want to make decisions about how best to architecture the site (whether to use node.js with express as middleware, or whether using React to organize the project.)
```
Comment (jrob8577) [inline README.md]: I don't understand the commitment you are making to a product that will be shipped by Friday.  What will the system do? What are the features? What will you complete?  How will completeness will be evaluated?  How will quality be evaluated?

Please enumerate the features you will be delivering, as well as the quality metrics that may be used for evaluation of completion and quality next Monday.

**Comment by alfonsotech** [comment]
Hi JRob, I updated README to include some quality metrics. I'm not sure I know what it is that you are missing from the outline, so if you could be more specific or stop by and chat with me about this, it would be greatly appreciated.

File: README.md [hunk 2]
```diff
@@ -28,3 +30,12 @@ This is a refactor for a project created on the front end. I would like to add a
 - [ ] User can take notes after being prompted by timer
 - [ ] User can rate notes
 - [ ] ... more
```
Comment (jrob8577) [inline README.md]: This should be omitted, or specified.

File: README.md [hunk 3]
```diff
@@ -28,3 +30,12 @@ This is a refactor for a project created on the front end. I would like to add a
 - [ ] User can take notes after being prompted by timer
```
Comment (jrob8577) [inline README.md]: Are these notes individual?  Shareable?  If so, authentication is probably a requirement that needs to be added to this list.

File: README.md [hunk 4]
```diff
@@ -28,3 +30,12 @@ This is a refactor for a project created on the front end. I would like to add a
...
 - [ ] User can rate notes
 - [ ] ... more 
+
+## Quality Rubric
+
+_What are some appropriate quality objectives for this goal? These are statements about the internal characteristics of the product that demonstrate fine design and craftspersonship, not its external features._
+
+- Quality rubric one: Three page website (a landing page, a curriculum map page, and an view-resource page): 60 point value
```
Comment (jrob8577) [inline README.md]: This feels like a completeness metric, not a quality metric. (The landing page, curriculum map page, and view resource page are features.)

File: README.md [hunk 5]
```diff
@@ -28,3 +30,12 @@ This is a refactor for a project created on the front end. I would like to add a
...
+- Quality rubric two: SQL Database with CRUD capabilities: 20 point value
```
Comment (jrob8577) [inline README.md]: This also feels like a completeness metric.  As well, as an external evaluator, how will I determine if this is completed?  Is one endpoint sufficient to meet this requirement?  Is there a best practice quality metric that can be added here (e.g. All endpoints in the system are defined as RESTful routes for each entity in our domain.)

I would also consider breaking the database requirement apart from the CRUD capabilities - they are distinct requirements (if you don't manage to complete all of the CRUD work, but do have a backing database, should you get zero completion for this metric?).

File: README.md [hunk 6]
```diff
@@ -28,3 +30,12 @@ This is a refactor for a project created on the front end. I would like to add a
...
+- Quality rubric three: Prepare repo for contributions as an Open Source project (w/ a project wiki): 10 point value
```
Comment (jrob8577) [inline README.md]: This is also a feature - you could restate this as a user story: As a contributor, I would like a README documenting the contribution guidelines for this project.

File: README.md [hunk 7]
```diff
@@ -28,3 +30,12 @@ This is a refactor for a project created on the front end. I would like to add a
...
+- Quality rubric four: Wireframe for website scale-ability: 10 point value
```
Comment (jrob8577) [inline README.md]: I don't understand this metric - how does wireframing relate to website scale?

**Pull request merged**